hack/e2e-gce/gcloud_create_cluster.sh

```diff
@@ -25,8 +25,8 @@ create_cluster() {
 	gcloud compute instances create descheduler-$node1_uuid --image="ubuntu-1604-xenial-v20180306" --image-project="ubuntu-os-cloud" --zone=us-east1-b
 	echo "gcloud compute instances delete descheduler-$node1_uuid --zone=us-east1-b --quiet" >> $E2E_GCE_HOME/delete_cluster.sh
 
-	gcloud compute instances create descheduler-$node2_uuid --image="ubuntu-1604-xenial-v20180306" --image-project="ubuntu-os-cloud" --zone=us-east1-b
-	echo "gcloud compute instances delete descheduler-$node2_uuid --zone=us-east1-b --quiet" >> $E2E_GCE_HOME/delete_cluster.sh
+	gcloud compute instances create descheduler-$node2_uuid --image="ubuntu-1604-xenial-v20180306" --image-project="ubuntu-os-cloud" --zone=us-east1-c
```

Comment:
Are these necessary? I believe the GCE "zones" are different from k8s topology-defined zones

Reply:
How are the e2e-gce tests ran today? Is that part of any pipeline?

Reply:
I believe these scripts are what set up the GCE clusters, but I'm referring to the changes in https://github.com/kubernetes-sigs/descheduler/pull/597/commits/82132e7b8b0add0030d85fd05fb5b216507583b5 to `Update gce scripts to spread nodes over 2 zones`

Reply:
The e2e tests currently relies on 2 unique zones being set on the nodes.

As to your question, I don't have an easy way of testing how this works when using kubeadm on GCE nodes because I don't see this GCE cluster being used anywhere. The e2e tests as part of the pipeline are ran on a kind cluster.
Moreover, I know that GKE uses the zone as the value for `topology.kubernetes.io/zone`